service/tbc/crawler.go

```diff
@@ -67,6 +68,56 @@ func (s *Server) TxIndexHash(ctx context.Context) (*HashHeight, error) {
 	return s.mdHashHeight(ctx, TxIndexHashKey)
 }
 
+func (s *Server) findCommonParent(ctx context.Context, bhX, bhY *tbcd.BlockHeader) (*tbcd.BlockHeader, error) {
```

Comment:
if this isn't used, let's leave it out of this pr

Reply:
It is now.